librz/core/cmd/cmd_print.c

```diff
@@ -1073,13 +1073,12 @@ static void print_format_help_help_help_help(RzCore *core) {
 static void cmd_print_fromage(RzCore *core, const char *input, const ut8 *data, int size) {
 	switch (*input) {
 	case 'a': {
-		asn1_setformat(input[1] != 'q');
-		RASN1Object *asn1 = rz_asn1_create_object(data, size);
+		RzASN1Object *asn1 = rz_asn1_object_parse(data, size);
 		if (asn1) {
-			char *res = rz_asn1_to_string(asn1, 0, NULL);
-			rz_asn1_free_object(asn1);
+			char *res = rz_asn1_to_string2(asn1, 0, input[1] != 'q');
+			rz_asn1_object_free(asn1);
 			if (res) {
-				rz_cons_printf("%s\n", res);
+				rz_cons_printf("%s", res);
```

Comment:
Is this an intended change?

Reply:
yes, before it was adding an unnecessary newline